test-image/test_transformer_class.py

```diff
@@ -0,0 +1,89 @@
+"""Tests for transformer_class.py
+"""
+# pylint:disable=import-error
+# pylint:disable=unused-import
```

Comment:
The `unused-import` error shouldn't need to be disabled - just remove or comment out the import statements

Reply:
I removed it, I'm not sure why I had it there to begin with, I might have been testing something and forgoten to remove it. I also removed the other disable as well.